README.md

```diff
@@ -81,6 +82,22 @@ Please replace `<VERSION>` with the actual version you want to install from the
 
 To verify that everything works you can visit Bridge, select a project, go to Uniform, and verify that `job-executor-service`  is registered as "remote execution plane" with the correct version and event type.
 
+**Example (with auto-detection)**
+
+For easier installation of the *job-executor-service* a single Keptn installation can be auto-detected. This only
+works if no `remoteControlPlane.api.token`, `remoteControlPlane.api.protocol` or `remoteControlPlane.api.hostname` is provided.
+If multiple Keptn installations are present, the `remoteControlPlane.autoDetect.namespace` must be set to the desired Keptn instance.
```

Comment:
If you leave it empty, it defaults to `keptn`?

Reply:
Empty means scan all existing namespaces, a namespace means just have a look at that one.